modules/@ergonode/attributes/src/config/extends.js

```diff
@@ -10,4 +103,158 @@ export default {
             requestPath: '/dictionary/attributes/types',
         },
     ],
+    extendStore: {
+        attribute: Store.Attribute,
+    },
+    extendMethods: {
+        '@Attributes/store/attribute/action/createAttribute/__before': ({
+            $this, type,
+        }) => {
+            const typeConfig = getTypeConfiguration({
+                $this,
+                type,
+            });
+
+            if (typeConfig) {
+                switch (type) {
+                case 'DATE':
+                case 'UNIT':
+                case 'PRICE':
+                    return prepareParametersData({
+                        $this,
+                        typeConfig,
+                    });
+                case 'TEXT_AREA':
+                    return prepareTextAreaData({
+                        $this,
+                        typeConfig,
+                    });
+                case 'SELECT':
+                case 'MULTI_SELECT':
+                    return prepareOptionsData({
+                        $this,
+                        typeConfig,
+                    });
+                default:
+                    return {};
+                }
+            }
+            return {};
+        },
+        '@Attributes/store/attribute/action/createAttribute/__after': ({
+            $this, data, type,
+        }) => {
+            const typeConfig = getTypeConfiguration({
+                $this,
+                type,
+            });
+
+            if (typeConfig) {
+                switch (type) {
+                case 'SELECT':
+                case 'MULTI_SELECT':
+                    return sendOptionsData({
+                        $this,
+                        typeConfig,
+                        data,
+                    });
+                default:
+                    return {};
+                }
+            }
+            return {};
+        },
+        '@Attributes/store/attribute/action/getAttribute/__after': ({
+            $this, data, type,
+        }) => {
+            const typeConfig = getTypeConfiguration({
+                $this,
+                type,
+            });
+
+            if (typeConfig) {
+                switch (type) {
+                case 'DATE':
+                case 'UNIT':
```

Comment:
I would consider to move unit attribute to Unit module

Reply:
no, if we move Unit we should move all types to modules.